pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_pantheon.adoc

```diff
@@ -0,0 +1,36 @@
+[id='about-pantheon_{context}']
+= About Pantheon
+
+Pantheon is a content-management system and publishing platform developed by Red Hat.
+You can use it to publish documents based on AsciiDoc source.
+After adding your modules and assemblies to Pantheon, you can do the following:
+
+. Add metadata
+. Preview your content using the look and feel of the Customer Portal
+. Associate modules and assemblies with a product
+. Publish, and unpublish content
+
+The following table describes features that Pantheon does and does not provide.
+
+.Pantheon Features
```

Comment:
Is a table the best way to represent this information? These feel more like lists than table contents to me.

Reply:
I had two lists. This just looks cleaner and  easier to parse to me. 6 of 1, half a dozen of the other. If you feel strongly, I'll change it back to lists.